pages/_resources/usgs-census-collaborate-roads-boundaries-data

```diff
@@ -0,0 +1,76 @@
+---
+resource_name: U.S. Geological Survey and U.S. Census Bureau collaborate on national roads and boundaries data
+slug: usgs-census-collaborate-roads-boundaries-data
+description: "It is a well-kept secret that the U.S. Geological Survey and the U.S. Census Bureau were the original two federal agencies to build the first national digital database of roads and boundaries in the United States. The agencies joined forces to develop homegrown computer software and state of the art technologies to convert existing USGS topographic maps of the nation to the points, lines, and polygons that fueled early GIS. Today, the USGS and Census Bureau have a longstanding goal to leverage and use roads and authoritative boundary datasets."
+source: U.S. Geological Survey and U.S. Census Bureau
+category: Case studies & examples
+tags:
+  - data cleaning
+  - data converter
```

Comment:
@BEA-RyanByrnes @rebeccawilliams it looks like the build is failing because this tag/keyword does not exist. You can either remove it, or define it in https://github.com/BEA-RyanByrnes/resources.data.gov/tree/patch-1/pages/_keywords